juju/juju
# Support migration of unit resources

Units may have different resource revisions to the application and as such need to be migrated separately.

This PR adds API support for unit resources and updates the binaries migration functionality to support them.

### QA

* Migrate a resource using model and confirm the resources collection matches up such that the unit resource is recorded. Also checked gridfs and blobstore metadata.
* Migrate a resource using model where the application resource is different to a unit resource and inspect the DB is correct post-migration. Run add-unit on the application.
